release_notes/topics/7_0_0.adoc

```diff
@@ -1,3 +1,7 @@
 = Apache Tomcat Adapter
 
 Java adapter for Apache Tomcat 8 and Apache Tomcat 9 was unified and now it serves for both of them.
+
+= Resolved "You took too long to login" issue
+
+The issue which occurred after first login request after a user session timeout has been resolved. 
```

Comment:
Sometimes, entering valid login credentials right away after login form was displayed resulted in _You took too long to login_ error instead of continuing login flow. This has now been fixed.